org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/program/Program.d

```diff
@@ -990,7 +665,168 @@ public override hash_t toHash() {
  * @return a string representation of the program
  */
 override
-public String toString() {
+public String toString() const {
     return Format( "Program {{{}}", name );
 }
 }
+
+@("test_equalsLjava_lang_Object")
+unittest
```

Comment:
I know the idiomatic way in D is to have the unit tests inline the same file. But to make the code closer to the original Java code it might be better to have them in the same place as the Java code.

Reply:
So move them to  `/tests/CLASSNAME.d`?  or would you prefer to mimic SWT? (`/tests/org.eclipse.swt.tests/CLASSNAME.d`)

Reply:
Put them in `/tests`. We're not following the directory structure of the SWT project anyway.